modules/user/src/main/ui/UserList.scala

```diff
@@ -88,22 +88,45 @@ final class UserList(helpers: Helpers, bits: UserBits):
         ))
     )
 
-  def top(perfType: PerfType, users: List[LightPerf])(using ctx: Context) =
-    val title = s"${perfType.trans} top 200"
+  def top(perfType: PerfType, users: List[LightPerf], page: Int, perPage: Int)(using ctx: Context) =
+    val from = (page - 1) * perPage + 1
+    val to = from + users.size - 1
+    val title = s"${perfType.trans} top $from–$to"
     Page(title)
       .css("bits.slist")
       .graph(
         title = s"Leaderboard of ${perfType.trans}",
-        url = s"$netBaseUrl${routes.User.topNb(200, perfType.key).url}",
-        description = s"The 200 best chess players in ${perfType.trans}, sorted by rating"
+        url = s"$netBaseUrl${routes.User.top(perfType.key).url}?page=$page",
+        description = s"The top rated players in ${perfType.trans}, sorted by rating ($from–$to)"
       ):
         main(cls := "page-small box")(
-          boxTop(h1(a(href := routes.User.list, dataIcon := Icon.LessThan), title)),
+          boxTop(
+            h1(a(href := routes.User.list, dataIcon := Icon.LessThan), title),
+            form(
+              cls := "pager",
+              action := routes.User.top(perfType.key).url,
+              method := "GET"
+            )(
+              input(
+                id := "page",
+                name := "page",
+                tpe := "number",
+                min := 1,
+                placeholder := "Page",
+                cls := "pager__input",
+                attr("inputmode") := "numeric"
+              )
+            )
```

Comment:
we already have dozens of paginators on lichess, that's not how they're done. Some examples:

- infinite scroll:
  - https://lichess.org/team/all
  - https://lichess.org/study
  - https://lichess.org/forum/general-chess-discussion
- pagination links:
  - https://lichess.org/forum/general-chess-discussion/what-are-the-best-openings-